src/sagas/index.js

```diff
@@ -254,11 +255,20 @@ function* refreshConsolePage ({ id }) {
   }
 }
 
+function* refreshVmSettingsPage ({ id }) {
+  const ids = id.split('/')
+  for (let vmId of ids) {
+    yield selectVmDetail(actionSelectVmDetail({ vmId }))
+  }
+}
+
```

Comment:
What happens if 60 VMs are multiselected on the VM setting page?  60 fetches every minute?  I'm not sure the VMs need to be updated like that at all while viewing the settings page.

Reply:
Fixed